lib/node/runner/checker_ballot_transaction.go

```diff
@@ -240,16 +245,27 @@ func ValidateTx(st *storage.LevelDBBackend, tx transaction.Transaction) (err err
 		return
 	}
 
+	var opsHasFee int
+	var hasFee bool
 	for _, op := range tx.B.Operations {
-		if err = ValidateOp(st, ba, op); err != nil {
-
+		if hasFee, err = ValidateOp(st, ba, op); err != nil {
```

Comment:
I think, `ValidateOp` and `hasFee` is quite different feature, so it should be split.